### PR TITLE
added aria, lint

### PR DESCRIPTION
**Pull request by aaarcher-usgs**
Added accessibility features that work with a screenreader and with keyboard control. Also ran `npm run lint` to clean up code.

The main approach to screen reader features is that the navigation buttons read the text for that section. I tested this on Safari (slight bug, see note) and Firefox with Mac's VoiceOver feature. I closed my eyes and ran through entire website with Firefox & Voiceover, and it worked very well. I have also used keyboard navigation without VO to test functionality on Firefox and on Safari. 

There is a known bug with Safari and Voiceover that it still reads components that are `aria-hidden="true"` even though it shouldn't. I mitigated this by changing the way in which the scroll trigger class change works, so that now the frames are missing (display = none) when they are "hidden" and not just "visibility = hidden". This is the last solution from this https://stackoverflow.com/questions/38772442/css-transition-from-display-none-to-display-block-navigation-with-subnav 

This solution also helps for screenreader that can still read components that are "visibility hidden"

To test, `nvm use 14.17.0` and `npm run serve`

### CODE REVIEW

File: src/components/DroughtThresholds.vue
Comment (hcorson-dosch-usgs) [inline src/components/DroughtThresholds.vue:2602]: Nice how quick this was to add!

**Review by hcorson-dosch-usgs — CHANGES_REQUESTED** [review]
Nice edits, Althea, and good catch on the issue w/ visibility.

### keyboard navigation

tested in Chrome, Edge, and Firefox

* Keyboard navigation works really well, though getting it started is a bit odd, I think because the buttons are initially hidden. When tab navigating, first tab takes you to the first link in the references, then backwards shift+tabs take you to the navigation buttons _unless_ upon page load you first hold the arrow key to scroll down to the first pane of the viz (which makes the buttons appear). I'm not 100% what keys are used in normal keyboard navigation. *Edit*: reading more I think arrow key to scroll to start navigation is just fine.
* Arrow key navigation w/ the down arrow progresses nicely, though you have to press it an inconsistent # of times to get to the next view (probs b/c of the scroll div heights and oddities of that approach). Seems like arrow keys are typically used for scrolling [here](https://webaim.org/techniques/keyboard/)
* I wonder if we could set it up so that the buttons could be tabbed to immediately. If I change the initial class setting for the `quietCircle` buttons to remove the `hidden` class it almost works, except that the first time you hit tab it scrolls to the second view, not the first (in Chrome and Edge - goes to first view in Firefox). Could be worth exploring more. Seems like the circles are there on page load, but pushed below the view until you start to scroll:

image

### keyboard navigation
For screen reader navigation, I tested this with windows Narrator on Edge, since that's the main supported browser

* With the current settings, with capslock + right arrow key navigation [per here](https://www.tpgi.com/basic-screen-reader-commands-for-accessibility-testing/), it reads the title, then skips content and goes to the References section
* When I had removed `hidden` from the initial class settings for the `quietCircle` buttons,  the same capslock + right arrow navigation reads the title, then starts in on content, reading the button text. Only hitches are a) it starts by showing the 5th visual (but reading from the beginning), and b) sometimes if I hit enter to navigate to the next visual, the button text isn't read. If the person does have some vision, they could use tab keys to back navigate and then again navigate forward to hear the text. If they don't have vision, it doesn't seem like an issue if the visuals don't progress with the text.

### Requested change/exploration

If you could do your own (time-boxed) exploration of allowing the `quietCircle` circle buttons to be visible on page load, and see how that affects page display, keyboard navigation and screenreader navigation on Mac, that would be great. It may not be worth implementing if a) it causes issues on mac, or b) we can't get it to show the first view first, but I think it's worth exploring given the benefits to keyboard and, especially, screenreader navigation on Windows.

*Edit to add:* Just chatted w/ Althea about this in Teams and we're both thinking the height of the scroll divs has to do w/ the view behavior and also #61

### Side comment

As a side note - For lint, I think it's nice to commit your work _then_ run lint and commit those changes separately. That way the reviewer can peruse your earlier substantive changes w/o wading through all the minor lint edits to formatting.

**Comment by aaarcher-usgs** [comment]
I agree that making the quiet circles visible on load makes it easier to navigate to the buttons. I committed that change and added another reference.

**Comment by aaarcher-usgs** [comment]
I am also seeing the issue of the viz jumping to button 2 frame visibly, even though it reads button 1. You can just select button 1 with keyboard navigation, but it's not making sense to me that it's jumping down to frame 2 other than through the div heights and the start/end locations of the scroll settings.

**Review by hcorson-dosch-usgs — APPROVED** [review]
Pulled down your changes, and it all works on my end! Good to merge once those merge conflicts are resolved. woohoo! 🎉

**Pull request merged**
